fix bug when Tests were grouping under Test_Name other than under Test_Suite_Name

gamerka opened the pull request:
Found a bug again, when test with name Ora() with several Test Cases were grouping under [assembly].[test_suite].Ora other then under [assembly].[test_suite]
Was caused by using Contains() method when finding a type in the assembly with the name of the test. In my case for Ora test it found some *Oracle* class name and marked whole [assembly].[test_suite].Ora as a separate test suite.
I think Equals() is more appropriate here.

gamerka committed:
fix bug when Tests were grouping under Test_Name other than under Test_Suite_Name

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/allure-framework/allure-nunit2?pullRequest=25) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/allure-framework/allure-nunit2?pullRequest=25) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/allure-framework/allure-nunit2?pullRequest=25) it.</sub>

gamerka committed:
+ Добавление в отчет для теста аттачей по путям, заранее заданным в свойствах теста, например:
[Property("AllureAttachment", @"..\..\WebTests\screen\OpenCaptureStationLink.png,C:\result.xml")]
